File: sample/ScaffoldingSample.TypeScript/Models/OrderDetail.ts
import { Order } from './Order';
import { Product } from './Product';

export interface OrderDetail {
    orderDetailId: number;
    orderId: number;
    productId: number;
    unitPrice: number;
    quantity: number;
    discount: number;
    order: Order;
    product: Product;
}

File: sample/ScaffoldingSample/Models/OrderDetail.cs
﻿using System;
using System.Collections.Generic; // Comment

namespace ScaffoldingSample.Models
{ // Comment
    public partial class OrderDetail // My Handlebars Helper
    {
        public int OrderDetailId { get; set; }
        public int OrderId { get; set; }
        public int ProductId { get; set; }
        public decimal UnitPrice { get; set; }
        public short Quantity { get; set; }
        public float Discount { get; set; }

        public virtual Order Order { get; set; }
        public virtual Product Product { get; set; }

        // My Handlebars Block Helper: True
        // My Handlebars Block Helper: False
    }
}
